main.cpp

```diff
@@ -185,33 +185,33 @@
      std::cout << firstName << " " << lastName << " made a total of " << numOfTripsToDumpster << " trips to the dumpster" << std::endl;
 
  }
- void CoffeeShop::printIncomeFromArt()
+ void CoffeeShop::printIncomeFromArt() const
  {
      std::cout << "This has added funds to the total annual profit, which is now £" << this->annualProfit  << std::endl;
  }
 
- float CoffeeShop::hostLocalArtistOnWalls(float percenatgeCommissionAgreed, float totalArtSales, std::string name)
+ float CoffeeShop::hostLocalArtistOnWalls(const float percenatgeCommissionAgreed, const float totalArtSales, const std::string name) const
  {
      float commissionTotal = percenatgeCommissionAgreed * totalArtSales;
      std::cout << name << " commission total: £" << commissionTotal << std::endl;
      return commissionTotal;
  }
 
- float CoffeeShop::chargeCustomer(int numberOfCoffees, std::string customerName)
+ float CoffeeShop::chargeCustomer(const int numberOfCoffees, const std::string customerName) const
  {
      float saleTotal = priceCoffee * numberOfCoffees;
      std::cout << customerName << " paid £" << saleTotal << std::endl;
      std::cout << "Each coffee from coffeeShop cost £" << this->priceCoffee << std::endl;
      return saleTotal;
  }
 
- bool CoffeeShop::trainStaff(Employee employee, std::string trainingRequired)
+ bool CoffeeShop::trainStaff(Employee& employee, std::string trainingRequired) const
  {
      std::cout << employee.firstName << " " << employee.lastName << " has been trained on " << trainingRequired << std::endl;
      return true;
  }
```

Comment:
`employee` is not being modified in this function
make it `const`
- [ ] revise